src/Tribe/Views/V2/Template/Page.php

```diff
@@ -46,11 +46,18 @@ public function get_path() {
 		} elseif ( 'page' === $template ) {
 			// We check for page as default is converted to page in Tribe\Events\Views\V2\Template_Bootstrap->in get_template_setting().
 			$template = get_page_template();
+
+			// TwentyTwenty does not have a Page Template...
+			if ( empty( $template ) ) {
+				$template = get_index_template();
+			}
 		} else {
 			// Admin setting set to a custom template.
 			$template = locate_template( $template );
 		}
 
+		// Maybe Check here instead for empty template and if empty try get_index_template()
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 609b85a3827f3f8cc91d1f2d48a946c1</sub>